contracts/DCAStrategies/DCAStrategies/DCAStrategiesManagementHandler.sol

```diff
@@ -4,18 +4,45 @@ pragma solidity >=0.8.7 <0.9.0;
 import '../../interfaces/IDCAStrategies.sol';
 
 abstract contract DCAStrategiesManagementHandler is IDCAStrategiesManagementHandler {
+  mapping(uint80 => Strategy) internal _strategies;
+  mapping(string => uint80) internal _strategyNames;
+  mapping(bytes32 => IDCAStrategies.ShareOfToken) internal _tokenShares;
+  uint80 public strategyCounter; // NOTE: could we use _mintCounter from DCAStrategiesPermissionsHandler?
+
+  function _checkTokenSharesSanity(IDCAStrategies.ShareOfToken memory _tokens) internal pure returns (bool) {
+    // TODO
+    return true;
+  }
+
+  function _getTokenSharesKey(uint256 _strategyId, uint80 _version) internal pure returns (bytes32) {
+    // NOTE:
+    return keccak256(abi.encodePacked(_strategyId, _version));
+  }
+
   /// @inheritdoc IDCAStrategiesManagementHandler
-  function getStrategy(uint80 _strategyId) external view override returns (IDCAStrategies.Strategy memory) {}
+  function getStrategy(uint80 _strategyId) external view virtual override returns (IDCAStrategies.Strategy memory) {}
```

Comment:
I think it would be better if we implemented this function instead of making it virtual and implementing it on the tests

It makes sense to implement it if it helps with testing